doc/api/util.md

```diff
@@ -536,6 +536,361 @@ added: v8.0.0
 A Symbol that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
+### Class: util.TextDecoder
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of the [WHATWG Encoding Standard][] `TextDecoder` API.
+
+```js
+const decoder = new TextDecoder('shift_jis');
+let string = '';
+let buffer;
+while (buffer = getNextChunkSomehow()) {
+  string += decoder.decode(buffer, { stream: true });
+}
+string += decoder.decode(); // end-of-stream
+```
+
+#### WHATWG Supported Encodings
+
+Per the [WHATWG Encoding Standard][], the encodings supported by the
+`TextDecoder` API are outlined in the table below. For each encoding,
+one or more aliases may be used. Support for some encodings is enabled
+only when Node.js is using the full ICU data.
+
+<table>
+<tr><th>Encoding</th><th>Alias</th><th>Requires Full-ICU</th></tr>
+
+<tr><td rowspan="3"><code>'utf-8'</code></td><td><code>'unicode-1-1-utf-8'</code></td><td rowspan="3">No</td></tr>
+<tr><td><code>'utf8'</code></td></tr>
+<tr><td><code>'utf-8'</code></td></tr>
```

Comment:
The list looks a bit cluttered… I.e. it makes the rest of the docs harder to read on that page, as this creates a long block in the middle of it so that navigating on the page requires more scrolling now.

Three questions:
1) does the main name need to be repeated in the aliases?
2) are quotes `'` needed?
3) would placing the aliases on a single line make them less accessible?

This is a minor nit, though — this can land without fixing this.

An alternative way of uncluttering that while keeping it clear and comprehensive would be to move that list to a separate docs page.